middleware/decrypt.py

```diff
@@ -20,6 +20,8 @@
 
 logger = logging.getLogger(__name__)
 
+# pylint: disable=logging-fstring-interpolation
+
```

Comment:
FYI: I think you can disable that globally - although maybe you wanna do that in one of the next PRs.